.storybook/manager-head.html

```diff
@@ -0,0 +1,34 @@
+<style>
+  .sidebar-item {
```

Comment:
Instead of `!important`

```suggestion
  .sidebar-item.sidebar-item {
```